packages/kbn-dev-utils/src/kbn_node/get_script_source.ts

```diff
@@ -0,0 +1,92 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { platform } from 'os';
+import { REPO_ROOT } from '../repo_root';
+
+const SCRIPT_SOURCE_SH = `#!/bin/sh
```

Comment:
Why does this file have to be generated? What are we doing here which can't be done with bash alone?

Reply:
We are generating them in order to be able to do this `DIR=${REPO_ROOT}`